### DISCUSSION TITLE
Incomplete log of wsl --status and performance problems when running wsl --update without admin rights

### DISCUSSION THREAD

**Discussion by ValentinTwin1206** [microsoft/WSL#9637] [General]
Hi all, 

I am running Windows 10 Enterprise (Version 20H2 Build 19042.2486) with Windows-in version of WSL2. Unfortunately, I do not have privileged rights on my machine and accidentally ran the wsl --update command. When prompted for credentials, I canceled the operation.

Now, when I run the wsl --status command, it does not display the kernel version anymore. The output only shows the default version and WSL distribution:

`
PS C:\Users\Username> wsl --status
DefaultVersion : 2
DefaultWSLDistro : Ubuntu-20.04
`

Additionally, WSL has become very slow. I have tried deactivating Windows Features and removing the WSL kernel, but the issues persist even after reinstalling.

Thanks
Valentin

**Comment by zcobol** [comment]
@ValentinTwin1206 what version of WSL did you install? Latest versions show the info you posted above when running `wsl --status`. Try `wsl --version` instead for details.

Sample output on my box:

```
PS C:\Users\zcobol> wsl --status
Default Distribution: openSUSE-Tumbleweed
Default Version: 2

PS C:\Users\zcobol> wsl --version
WSL version: 1.1.2.0
Kernel version: 5.15.83.1
WSLg version: 1.0.49
MSRDC version: 1.2.3770
Direct3D version: 1.608.2-61064218
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.19045.2546
```

**Comment by ValentinTwin1206** [comment]
HI @zcobol as mentioned, I am using the "Windows-in" version, so I have not downloaded anything from the store but enabled the Windows feature. My WSL version does not support `--version` flag.
So far, my company does not allow the usage of the store version

Best Valentin
